src/ctia/schemas/core.clj

```diff
@@ -11,6 +11,26 @@
             [schema-tools.core :as st]
             [schema.core :as s :refer [Bool Str]]))
 
+(s/defschema APIHandlerServices
+  "Maps of services available to routes"
+  {:ConfigService {:get-config (s/=> s/Any s/Any)
```

Comment:
I reckon that you have considered this, but given s/=> is merely informative, we might as well discard this mechanism as it leads to false confidence, and can drift vs. reality